### ISSUE TITLE
Multiple paragraph spacings if newline character occur in text

### ISSUE THREAD

**Issue by vpetzel** [gitbrent/PptxGenJS#368]
When adding Text with multiple Lines and paragraph spacing informations like
```
<html>
  <head>
    <script lang="javascript" src="PptxGenJS/libs/jquery.min.js"></script>
    <script lang="javascript" src="PptxGenJS/libs/jszip.min.js"></script>
    <script lang="javascript" src="PptxGenJS/dist/pptxgen.js"></script>
  </head>
  <body>
    <script>
        pptx = new PptxGenJS();
        var slide = pptx.addNewSlide();

        slide.addText("SOME TEXT\nSOME OTHER TEXT\nTHIRD TEXT", {x:'0', y:'0', w:'100%', h:'100%', align:'center', valign:'middle', paraSpaceAfter:2.56, margin: 0});

//          Save PPTX
        pptx.save('Titel');
    </script>
  </body>
</html>;
```
the corresponding slide.xml in ./slides/ has a rising number of
```
<a:spcAft>
<a:spcPts val="[whatever]"/>
</a:spcAft>
```
statements, that is, before the first line you get one statement, before the second two, ...
This happens because in pptxgen.js the variable strXmlParaSpc does not get reset after completing one run of the containing loop in line 3609.
Here’s a small patch for the issue:
[pptxgen.js.patch.txt](https://github.com/gitbrent/PptxGenJS/files/2148495/pptxgen.js.patch.txt)

**Comment by vpetzel** [comment]
By the way, this leads to the file being handled as currupted by MS Powerpoint.

**Comment by gitbrent** [comment]
Thanks @vpetzel !

Patch accepted and included in upcoming v2.3.0

**Comment by frappierer** [comment]
@gitbrent 
We have the Problem with the corrupted PPTX files as well (Because of the line breaks). Do you know when the new release will come?

By the way, the software is amazing!
